sql/parser2/scan_test.go

```diff
@@ -40,11 +40,9 @@ func TestScanner(t *testing.T) {
 		{`>`, []int{'>'}},
 		{`>=`, []int{GREATER_EQUALS}},
 		{`=`, []int{'='}},
-		{`=>`, []int{EQUALS_GREATER}},
```

Comment:
Is => not used as a comparison operator too, though?
On Jul 17, 2015 6:34 PM, "Peter Mattis" <notifications@github.com> wrote:

> In sql/parser2/scan_test.go
> <https://github.com/cockroachdb/cockroach/pull/1720#discussion_r34936851>:
>
> > @@ -40,11 +40,9 @@ func TestScanner(t *testing.T) {
> >  		{`>`, []int{'>'}},
> >  		{`>=`, []int{GREATER_EQUALS}},
> >  		{`=`, []int{'='}},
> > -		{`=>`, []int{EQUALS_GREATER}},
>
> => and := were both allowed by the postgres grammar for the specification
> of named arguments to custom functions. This wasn't mentioned in the docs,
> but can be seen from the original grammar:
> https://github.com/postgres/postgres/blob/master/src/backend/parser/gram.y#L12936
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cockroachdb/cockroach/pull/1720/files#r34936851>.
>